sdk/storage/Azure.Storage.Blobs.Batch/api/Azure.Storage.Blobs.Batch.net6.0.cs

```diff
@@ -0,0 +1,33 @@
+namespace Azure.Storage.Blobs.Specialized
```

Comment:
Did you mean to include these storage changes?

Reply:
I don't think these changes should be related to the credential changes. Perhaps your local branch is out of sync with main?